New package: Hanzi v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: Hanzi
- Repository: https://github.com/NicoACloutier/Hanzi.jl
- Created by: @NicoACloutier
- Version: v1.0.0
- Commit: 30ea7b176df9f834133eccecf276bef3150a1993
- Reviewed by: @NicoACloutier
- Reference: https://github.com/NicoACloutier/Hanzi.jl/issues/1#issuecomment-2019104258
- Description: A Julia library to romanize Hanzi.
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a8359823aaa70bdf07ded43d05067c478ce515acaa5c847227c1410d3e8eb12652cf48c675f019d5dc1e5a9654799747d21fa5d5804ba49283e0fcf216b29fc2c262faf7f0b245187cf51e235c0ea55d6b9db2b75063e751390866cb4a7b04e5d1d6b6c86088e10efcf12e20c78ced804b4a88c5bc1c224e9d496f97505de0960fe215 -->

goerz commented:
[noblock] The name similarity is no problem, but you have to fix whatever is preventing the package from loading. You can resubmit the registration without changing the version number, and it will update this existing PR.

JuliaRegistrator committed:
New package: Hanzi v1.0.0

UUID: 485a87e4-fb71-42ca-b614-bc87b1bb3702
Repo: https://github.com/NicoACloutier/Hanzi.jl.git
Tree: c3689c459bf5aedc8cd5df59489b95640c2f16f8

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
